### ISSUE TITLE
Old build artifacts not removed from s3

### ISSUE THREAD

**Issue by yzq89** [serverless-components/website#66]
Hi there, first off thanks for making such an easy to use framework!

I'm trying out this component for deploying a React SPA with S3 & CloudFront. However I'm noticing that build artifacts from previous builds remain in my bucket - is there an option to remove these on a fresh deploy? (Similar to the default behaviour of serverless-finch)

Best
Zac

**Comment by BrunoBernardino** [comment]
Hi @zac-yang did you end up using something else or building your own custom script to deploy? I'm starting to evaluate that option, but would prefer not to, really.

**Comment by zac-yang** [comment]
Nope, I didn't end up resolving this issue, and am not actively using this currently.

**Comment by BrunoBernardino** [comment]
Thanks.
